settings.gradle

```diff
@@ -28,6 +28,6 @@ include 'cruise-control', 'cruise-control-metrics-reporter', 'cruise-control-cor
 def gradleVer = GradleVersion.current()
 def minimumVersion = GradleVersion.version("3.5")
 if (gradleVer < minimumVersion) {
-  throw new GradleScriptException("this build cannot be run with gradle < " + minimumVersion + ". current detection version is "
-      + gradleVer + ". use newer gradle or (better yet) use the wrapper", null)
+  throw new GradleScriptException("this build cannot be run with gradle < " + minimumVersion + ". current detection version is " +
+      gradleVer + ". use newer gradle or (better yet) use the wrapper", null)
```

Comment:
```suggestion
  throw new GradleScriptException("this build cannot be run with gradle < " + minimumVersion + ". current detection version is "
                                  + gradleVer + ". use newer gradle or (better yet) use the wrapper", null)
```

Reply:
The original change regarding moving `+` to the previous line was intentional.

Contrary to Java, in multi-line Strings for Groovy the preferred style is to keep the operator on the previous line. Otherwise, the IDE will show a warning that this syntax is incorrect.

Source: https://stackoverflow.com/questions/5079797/whats-wrong-with-groovy-multi-line-string